lib/Common/CommonDefines.h

```diff
@@ -132,9 +132,10 @@
 // Language features
 #if !defined(CHAKRACORE_LITE) && (defined(_WIN32) || defined(INTL_ICU))
 #define ENABLE_INTL_OBJECT                          // Intl support
-#define ENABLE_JS_BUILTINS                          // Built In functions support
 #endif
 
+#define ENABLE_JS_BUILTINS                          // Built In functions support
```

Comment:
I assume it's historical, so that we had a way to back out if needed. Js Builtins are relatively new with the first one, indexOf, it was not immediately clear if the benefits were significant.